roottest/root/io/newClassDef/current/CMakeLists.txt

```diff
@@ -14,10 +14,16 @@ ROOTTEST_GENERATE_DICTIONARY(libIoNewClassnamespace
                              NO_CXXMODULE
                              FIXTURES_SETUP root-io-newClassDef-current-namespace-fixture)
 
+# there is no ClassDefT macro with override syntax, therefore suppress warning
```

Comment:
`ClassDefT` is obsolete.  You can used `ClassDefOverride` in its stead.

Reply:
But that about ClassDefT2?

Reply:
And may be we can keep old code - to test compatibility?

Reply:
Ok, lets try with `ClassDefOverride`

Reply:
It works! 
Was never aware about existing of such defines.